packages/docusaurus/bin/docusaurus.js

```diff
@@ -42,13 +42,11 @@ program
   .command('build [siteDir]')
   .description('Build website')
   .option(
-    '-sic, --skip-image-compression <skipImageCompression>',
```

Comment:
This is actually doing nothing now. Remove it for now (will add it when we port v1 to v2 command)

Reply:
I don't think we need to port this. Because we are not implementing our own image compression (unlike in v1), it should be more robust and we don't need to let people skip it. At least that's my opinion.

If not we're going to have a barrage of CLI options for very specific purposes. If people don't want it, they can choose to opt in/out of the webpack plugin via modifying it themselves.